scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/defaults/main.yml

```diff
@@ -20,7 +20,11 @@ ardana_qe_test_work_dir: "{{ ardana_qe_base_dir }}/{{ test_name }}"
 ardana_qe_tests_dir: "{{ ardana_qe_base_dir }}/ardana-qa-tests"
 ardana_qe_tests_git_repo: "http://git.suse.provo.cloud"
 ardana_qe_tests_git_url: "{{ ardana_qe_tests_git_repo }}/ardana/ardana-qa-tests.git"
-ardana_qe_tests_branch: "{{ tests_branch | default('master') }}"
+
+# cloudsource is not required for QA tests, so we set a 'dummy' cloudsource
+# value based on the SLES SP version
+cloudsource: "{{ (ansible_distribution_release == '3') | ternary('8', '9') }}"
```

Comment:
For this "hack" to work, the variable should always be evaluated in the context of the deployer, or one of the other nodes in the test cloud. Can you confirm that is the case ?

Reply:
Yes, for that case the context will always be the deployer which is where the test run.